### DISCUSSION TITLE
PaddleOCR总是把竖线符号【|】识别成数字【1】

### DISCUSSION THREAD

**Discussion by zealinux** [PaddlePaddle/PaddleOCR#12395] [Q&A]
- 版本号/Version：
   Paddle：paddlepaddle>=2.6.0
  PaddleOCR：paddleocr>=2.6.1.2

- 运行指令/Command Code：paddleocr --image_dir .

本来应该：是秒杀380mL/瓶｜味事达味
识别成了：秒杀380mL/瓶1味事达味

**Comment by tink2123** [comment]
这是比较难的case，如果场景固定建议通过后处理解决，如果不固定推荐自己训练数据finetune
